src/Streaming/Prelude/Linear.hs

```diff
@@ -24,9 +24,9 @@
 
    Other libraries that come up in passing are
 
-> import qualified Control.Foldl as L -- cabal install foldl
-> import qualified Pipes as P
-> import qualified Pipes.Prelude as P
+> import qualified Control.Foldl as Foldl -- cabal install foldl
+> import qualified Pipes
+> import qualified Pipes.Prelude as Pipes
```

Comment:
Don't do it here. You can do this but then you should change it consistently throughout the haddock which will take forever.

Reply:
I do really want single-letter imports gone everywhere. They are bad practice, and I don't want our examples to use encourage bad practices.

Reply:
Okay, but doing it throughout all the streaming docs that took me more than a day to read though isn't a good use of time right now. I think we should make an issues for post-release.

Reply:
Then I will undo this change, and keep the issue open, no need to make the Haddock changes hold up this PR.